src/controllers/group/index.js

```diff
@@ -30,7 +30,11 @@ export const getParticipantsCount = (obj, args, context) => {
 export const createGroup = async (obj, { input }, context) => {
   const productId = await createProductIfNotGiven(input, context.models)
 
-  return context.models.group.create({ ...input, productId })
+  return context.models.group.create({
+    ...input,
+    productId,
+    sellerId: context.currentUser && context.currentUser.id
```

Comment:
We shouldn't ask if the currentUser exists. This mutation can NOT be executed if there is no logged-in user. Let's wrap this resolver inside a requireAuth hoc ;)